src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -1233,6 +1433,20 @@ namespace Microsoft.Management.Deployment
         /// (Installing, Installed) together at the same time.
         PackageCatalogReference CreateCompositePackageCatalog(CreateCompositePackageCatalogOptions options);
 
+        [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 11)]
+        {
+            Windows.Foundation.IAsyncOperationWithProgress<AddPackageCatalogResult, AddPackageCatalogProgress> AddPackageCatalogAsync(AddPackageCatalogOptions options);
+
+            /// Removes a source from the Windows Package Manager source list.

```

Comment:
We should only refer to them as catalogs within the COM API.  If there are other cases where we use source, they should be changed (unless it is explicitly a call out on the different nouns being used).